src/main/java/com/netflix/imflibrary/st0377/header/Preface.java

```diff
@@ -93,6 +93,15 @@ public int getNumberOfEssenceContainerULs()
         return this.prefaceBO.essencecontainers.size();
     }
 
+    /**
+     * Getter for a list of UL of Specifications to which this MXF file complies
+     * @return list of UL of Specifications to which this MXF file complies
+     */
+    public CompoundDataTypes.MXFCollections.MXFCollection<UL> getConformstoSpecificationsULs()
```

Comment:
It might be a good idea to unpack this into a regular Java `List` and return that instead. The only place it is currently being used unpacks it immediately anyway. That would make it more consistent with other places where `MXFCollection` is a field.

Reply:
As discussed, we will not change that now.